controls/os_spec.rb

```diff
@@ -228,3 +228,19 @@
     it { should be_empty }
   end
 end
+
+control 'os-10' do
+  impact 1.0
+  title "CIS: Disable unused filesystems"
+  desc "1.1.1 Ensure mounting of cramfs, freevxfs, jffs2, hfs, hfsplus, squashfs, udf, FAT"
+  describe file('/etc/modprobe.d/CIS.conf') do
```

Comment:
@bitvijays I thought its already covered by you and you are aware of changes ;-)